lib/logHooks.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const async = require('async');
+
+const current = {
+  processing: 0,
+  waiting: 0,
+  completed: 0
+};
+
+module.exports = (collection, allDone) => {
```

Comment:
You call this log hooks, but it doesn't do any logging. Should be called hookStats or something